CHANGELOG.md

```diff
@@ -10,7 +10,71 @@ The format is based on [Keep a Changelog], and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- A new mechanisim for instantiating a pagerduty instance ([#64]).
+
+  ```ruby
+  pagerduty = Pagerduty.build(integration_key: "<my-integration-key>",
+                              api_version:     "1")
```

Comment:
API version as a string looks a little strange. I would expect to see this:

```suggestion
                              api_version:     2)
```

Reply:
👍 Both numeric and string values will work. I've added a test for both in 98765c7, and updated the documentation to use numeric values.